src/hotspot/share/gc/g1/g1_globals.hpp

```diff
@@ -168,11 +168,6 @@
           "Size of an update buffer")                                       \
           range(1, NOT_LP64(32*M) LP64_ONLY(1*G))                           \
                                                                             \
-  product(uintx, G1ConcRefinementServiceIntervalMillis, 300,                \
```

Comment:
Since this is a product option we need to add this flag as obsolete/removed in the CSR request as already suggested there in a comment.

I doubt that anybody ever used this one ever.

Reply:
Oops, forgot that.  I did remember to move the CSR back to draft, pending updating it for this change.